CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.10.6 (Next)
 
 * Your contribution here.
+* [#182](https://github.com/slack-ruby/slack-ruby-bot/pull/182): CommandsHelper class and Help module refactoring - [@mdudzinski](https://github.com/mdudzinski).
```

Comment:
Sounds better as `Refactored ...`

Reply:
Sure, I'll change it.

Reply:
Actually, it should be `Refactor ...` as commit title/summary should be imperative. Unless you want the pr's title to be different than the commit's one.